projects/js-packages/charts/src/components/shared/with-responsive.tsx

```diff
@@ -65,6 +65,7 @@ export function withResponsive< T extends Exclude< BaseChartProps< unknown >, 'o
 				style={ {
 					width: '100%',
 					height: chartProps.height ?? 'auto',
+					overflow: 'hidden',
```

Comment:
Unfortunately this causes custom annotations to be cut off if they're positioned near charts edges. Maybe the consumer could use margins to adjust the space around the chart, but why is the overflow clipping necessary?

![Screenshot 2025-07-23 at 1 15 17 PM](https://github.com/user-attachments/assets/72753908-9645-4a1f-a9c4-0ea8e55142cb)

Reply:
Pie chart also being clipped

![Screenshot 2025-07-23 at 2 43 53 PM](https://github.com/user-attachments/assets/6b68e152-08f6-4fc2-b098-9c47f634b023)

Reply:
Nice spotting!! Thank you for highlighting this. I sat on this PR for way too long and involved too many rebases, some things got a bit.. odd. I definitely missed this one and am fixing up asap 🙏